addons/sale_project/static/src/right_panel/project_right_panel.js

```diff
@@ -24,5 +25,11 @@ patch(ProjectRightPanel.prototype, '@sale_project/right_panel/project_right_pane
         if (saleItems && saleItems.total > saleItems.data.length) {
             await this._loadAdditionalSalesOrderItems();
         }
+    },
+
+    formatValue(value, unit) {
+        return unit === 'Hours'
+                      ? formatFloatTime(value)
+                      : formatFloat(value);
```

Comment:
You can write it in one line, since the line will not be very long. 🙂 
```suggestion
        return unit === 'Hours' ? formatFloatTime(value) : formatFloat(value);
```